MKHorizMenu/MKHorizMenu.m

```diff
@@ -69,14 +69,15 @@ -(void) reloadData
         customButton.tag = tag++;
         [customButton addTarget:self action:@selector(buttonTapped:) forControlEvents:UIControlEventTouchUpInside];
         
-        int buttonWidth = [title sizeWithFont:customButton.titleLabel.font
-                            constrainedToSize:CGSizeMake(150, 28) 
-                                lineBreakMode:UILineBreakModeClip].width;
+        CGRect buttonRect = [title boundingRectWithSize:CGSizeMake(150, 28)
+                                                options:NSStringDrawingUsesLineFragmentOrigin
+                                             attributes:@{NSFontAttributeName:customButton.titleLabel.font}
```

Comment:
Please also include the line break mode into attributes. See http://stackoverflow.com/questions/19197709/what-is-the-equivalent-of-nslinebreakmode-in-ios-7-attributed-strings-drawing-me for example.